client/src/components/DropZone.js

```diff
@@ -2,9 +2,15 @@ import React from 'react'
 import { Box, Text, Button } from 'grommet'
 import Dropzone from 'react-dropzone'
 
-export default ({ fileTypes = [], onDrop }) => {
+// We are restricted to 1GB attachments
+export default ({ fileTypes = [], multiple = true, onDrop }) => {
   return (
-    <Dropzone accept={fileTypes.map((ft) => `.${ft}`)} onDrop={onDrop}>
+    <Dropzone
+      multiple={multiple}
+      accept={fileTypes.map((ft) => `.${ft}`)}
+      onDrop={onDrop}
+      maxSize={1000000000}
```

Comment:
Pulling this out into a const would reduce magic numbers, even though its documented above.